rct229/data/ashrae_90_1_table_G3_5_3.json

```diff
@@ -0,0 +1,46 @@
+{
+  "prm_water_chiller_min_efficiency": [
+    {
+      "cooling_type": "displacement",
+      "minimum_capacity": 0.0,
+      "maximum_capacity": 149.99,
```

Comment:
Another way to express the strict inequality from the table would be to call the capacity fields "inclusive_minimum_capacity" and "exclusive_maximimum_capacity".